i18n/vscode-language-pack-uk-UA/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Change Log
+All notable changes to the "vscode-language-pack-ru" language pack will be documented in this file.
```

Comment:
This should say `language-pack-uk` (see also https://github.com/microsoft/vscode-loc/pull/1239#issuecomment-1632992235)